### PR TITLE
Improves and fixes inconsistencies in boundary conditions docs

### PR DESCRIPTION
**Pull request by glwagner**

### CODE REVIEW

File: docs/src/model_setup/boundary_conditions.md
Comment (navidcy) [inline docs/src/model_setup/boundary_conditions.md:44]:
```suggestion
   For example, sunlight absorbed at the ocean surface imparts a temperature flux that heats near-surface fluid.
```

File: docs/src/model_setup/boundary_conditions.md
Comment (navidcy) [inline docs/src/model_setup/boundary_conditions.md:45]:
```suggestion
   If there is a known `diffusivity`, you can express `FluxBoundaryCondition(flux)` using `GradientBoundaryCondition(-flux / diffusivity)` (aka "Neumann" boundary condition).
```

**Review by navidcy — APPROVED** [review]
Let's wait to see the Docs preview first!

**Comment by navidcy** [comment]
how about changing `!!! info "The flux convention in Oceananigans"` to `!!! warning "The flux convention in Oceananigans"`?

**Comment by glwagner** [comment]
> how about changing `!!! info "The flux convention in Oceananigans"` to `!!! warning "The flux convention in Oceananigans"`?

Ahaha I'm not sure it qualifies as a "warning" does it? Nothing is the matter, it's just info you need to know!

**Comment by glwagner** [comment]
Preview here: https://clima.github.io/OceananigansDocumentation/previews/PR1836/model_setup/boundary_conditions/

**Comment by navidcy** [comment]
@glwagner, shall I/u merge?

**Comment by navidcy** [comment]
I'm taking the liberty...

**Comment by glwagner** [comment]
Ah, was just waiting for approval that you were ok I didn't add the warning you requested!

**Comment by navidcy** [comment]
That was just a suggestion, not a requirement. ;)

**Pull request merged**
